src/planscape/conditions/raster_utils.py

```diff
@@ -15,17 +15,29 @@
 RASTER_NAME_COLUMN = 'name'
 
 
-# Returns np.nan if no entries are available.
-# This should be differentiated from None, which is a possible value if the
-# score was previously computed, but no intersection exists between a plan
-# geometry and the condition raster.
-# TODO: return enum values or status instead of None or np.nan.
-def _get_db_score_for_plan(plan_id, condition_id) -> float | None:
+# Statistics across stands within a subarea of a raster.
+class ConditionStatistics(TypedDict):
+    # Mean across stand values (sum / count)
+    mean: float
+    # Sum of stand values.
+    sum: float
+    # The number of stands counted.
+    count: int
+
+
+# Returns None if no statistics are available.
+# Otherwise, returns ConditionStatistics.
+def _get_db_stats_for_plan(
+        plan_id, condition_id) -> ConditionStatistics | None:
     db_scores = ConditionScores.objects.filter(
         plan_id=plan_id).filter(condition_id=condition_id).all()
     if len(db_scores) > 0:
-        return db_scores[0].mean_score
-    return np.nan
+        db_score = db_scores[0]
+        return ConditionStatistics(
```

Comment:
can we store ConditionStatistics directly in the db?

Reply:
I started working on making this return an instance of the ConditionScores model, but then realized we still need ConditionStatistics or something similar as an output to compute_condition_stats_from_raster, whose input is a geo and a raster name.

Not worth digging into for this pull request imo - more of a "polish" than "barebones" task.